src/renderer/redux/reducers/media.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import * as actions from "constants/action_types";
+import { handleActions } from "util/redux-utils";
+
+export type MediaState = {
+  paused: Boolean,
+  positions: Object,
```

Comment:
can you define a type more specifically than `Object`? Maybe `{ [string]: number }`?

https://flow.org/en/docs/types/objects/#toc-objects-as-maps